content/strings.yml

```diff
@@ -0,0 +1,48 @@
+"Babel could not be loaded": ""
+"This can be caused by an ad blocker. If you're using one, consider adding reactjs.org to the whitelist so the live code examples will work.": ""
+"Loading code example...": ""
+"When you encounter an error, you'll receive a link to this page for that specific error and we'll show you the full error text.": ""
+"The full text of the error you just encountered is:": ""
+"Edit this page": ""
+"We couldn't find what you were looking for.": ""
+"Page Not Found": ""
+"Please contact the owner of the site that linked you to the original URL and let them know their link is broken.": ""
+"Get Started": ""
+"Take the Tutorial": ""
+"A JavaScript library for building user interfaces": ""
+"The React documentation is available in the following languages:": ""
+"In Progress": ""
+"Needs Contributors": ""
+"Don't see your language above?": ""
+"Let us know": ""
+"Contribute": ""
+"Redirecting to Codepen...": ""
+"Not automatically redirecting?": ""
+"All posts ...": ""
+"Recent Posts": ""
+"Previous article": ""
+"Next article": ""
+"React - Acknowledgements": ""
+"We'd like to thank all of our contributors:": ""
+"In addition, we're grateful to": ""
+"<0>Jeff Barczewski</0> for allowing us to use the <1>react</1> package name on npm.": ""
+"<0>Christopher Aue</0> for letting us use the <1>reactjs.com</1> domain name and the <2>@reactjs</2> username on Twitter.": ""
+"<0>ProjectMoon</0> for letting us use the <1>flux</1> package name on npm.": ""
+"Shane Anderson for allowing us to use the <1>react</1> org on GitHub.": ""
+"<0>Dmitri Voronianski</0> for letting us use the <1>Oceanic Next</1> color scheme on this website.": ""
+"React - All Posts": ""
```

Comment:
We should change these page titles to be consistent with the way they're displayed for the Docs/Tutorial section (e.g. "Getting Started - React"). That way we can just have one string (the page title) and append React to the end.

Reply:
Helmet has titleTemplate prop; so, I utilized it to solve this issue. Now, there are two new strings:

    "titleTemplate": "%s – React"
    "blogTitleTemplate": "%s – React Blog"

I also added title template for <TitleAndMetaTags> component. This way, different pages can have different title templates. I kept it very flexible for simplicity. If you want to, I can keep the same template and add a postfix (like how it used to) that will change based Blog or other pages.

I kept "React – A Javascript Library..." for homepage.